resources/kubernetes/operator/helm/values.yaml

```diff
@@ -180,19 +212,32 @@ collectors:
             exporters:
             - debug
             - elasticsearch/otel
+
+  #  Daemon is a K8s daemonset EDOT collector focused on gathering telemetry at
+  #  node level and exposing an OTLP endpoint for data ingestion.
+  #  Auto-instrumentation SDKs will use this endpoint.
   daemon:
+    # Configure the pods resources to control CPU and memory usage.
+    resources:
+      limits:
+        cpu: 100m
+        memory: 500Mi
+      requests:
+        cpu: 100m
+        memory: 500Mi
     presets:
```

Comment:
Good catch, I commented this out because the deployment already has these resource values: https://github.com/open-telemetry/opentelemetry-helm-charts/blob/main/charts/opentelemetry-kube-stack/values.yaml#L519-L525
see also https://github.com/elastic/opentelemetry/pull/33#issuecomment-2437268504